tests/SshTest.php

```diff
@@ -149,4 +150,13 @@ public function it_can_configure_the_used_process()
 
         $this->assertMatchesSnapshot($command);
     }
+
+    /** @test */
+    public function it_can_run_a_command_locally()
+    {
+        $local = new Ssh('user', '127.0.0.1');
+        $command = $local->execute('whoami');
+
+        Assert::assertEquals(trim($command->getOutput()), get_current_user());
```

Comment:
Use `$this->assertEquals()`

Reply:
Updated this so that the code uses `$this->assertEquals()`